torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -721,7 +817,9 @@ void PeepholeOptimizeONNX(std::shared_ptr<Graph>& graph, int opset_version, bool
   fuseListConstructListUnpack(graph->block());
   fuseSplitListUnpack(graph->block());
   fuseUnbindListUnpack(graph->block());
```

Comment:
I am not sure to understand why we call fuseUnbindListUnpack() here and then again in convertUnbindToSplit(). Is it related to the eraseListConstruct() call?

Reply:
That's a mistake... Removing duplicated fuseUnbindListUnpack.